sdl_android/src/main/java/com/smartdevicelink/api/SdlManager.java

```diff
@@ -546,19 +546,19 @@ public void sendRPCs(List<? extends RPCMessage> rpcs, final OnMultipleRequestLis
 	}
 
 	/**
-	 * Add an OnRPCNotificationListener for HMI status notifications
-	 * @param listener listener that will be called when the HMI status changes
+	 * Add an OnRPCNotificationListener
+	 * @param listener listener that will be called when a notifications is received
```

Comment:
This should be: "listener that will be called when a **notification** is received"